recipes-tests/trenchboot-tests/trenchboot-tests/cukinia.conf

```diff
@@ -0,0 +1,74 @@
+export CUKINIA_ROOT=/etc/cukinia
+
+shopt -s nullglob
+
+check_acm() {
+    files=("/boot/acm/$1"*.bin)
+    if [ ${#files[@]} -eq 0 ]; then
+        return 1
+    fi
+    for acm in "${files[@]}"; do
+        if [ -f "$acm" ]; then
+            grep "/acm/$(basename "$1")" /boot/grub/grub.cfg || return 1
+        else
+            return 1
+        fi
+    done
+}
+
+acms=(
+    ADL_SINIT
+    BDW_SINIT
+    CFL_SINIT
+    CMLSTGP_SINIT
+    CML_RKL_S_SINIT
+    CML_S_SINIT
+    RKLS_SINIT
+    SKL_KBL_AML_SINIT
+    SNB_IVB_SINIT
+    TGL_SINIT
+)
+
+cukinia_log "$(_colorize yellow "--- GRUB ---")"
+cukinia_test -f /boot/grub/grub.cfg
+cukinia_test -f /boot/EFI/BOOT/grub.cfg
+as "Check if /boot/grub/grub.cfg and /boot/EFI/BOOT/grub.cfg are the same" \
+    cukinia_cmd cmp /boot/grub/grub.cfg /boot/EFI/BOOT/grub.cfg
+as "grub.cfg contains slaunch" cukinia_cmd grep 'slaunch\s*$' /boot/grub/grub.cfg
+
+cukinia_log "$(_colorize yellow "--- ACM & SKL ---")"
+for acm in "${acms[@]}"; do
+    as "/boot/acm/$acm*.bin exists and is used in grub.cfg" cukinia_cmd check_acm "$acm"
+done
+cukinia_test -f /boot/skl.bin
+as "skl.bin is used in grub.cfg" cukinia_cmd grep "/skl.bin" /boot/grub/grub.cfg
+
+cukinia_log "$(_colorize yellow "--- packages ---")"
+as "Check whether the tpm2_tools package can be started" cukinia_cmd tpm2_eventlog -v
+as "Check whether the rsync package can be started" cukinia_cmd rsync --version
+
+cukinia_log "$(_colorize yellow "--- TPM ---")"
+# cukinia_test -f /sys/kernel/security/tpm0/binary_bios_measurements
+# as "TPM event log can be read by tpm2_eventlog" \
+#     cukinia_cmd tpm2_eventlog /sys/kernel/security/tpm0/binary_bios_measurements
+
+pcrs=(/sys/class/tpm/tpm0/pcr-sha*)
+as "At least one PCR bank is active" cukinia_test ${#pcrs[@]} -gt 0
+for pcr_bank in "${pcrs[@]}"; do
+    as "$(basename "$pcr_bank") 0..23 can be read" cukinia_cmd cat "$pcr_bank"/{0..23}
+done
+
+for pcr_bank in "${pcrs[@]}"; do
```

Comment:
At this complexity level, I would probably consider separate tests in RF.
Also, it would not work in QEMU I assume?
And of the main goals of cukinia is to keep in running locally when developing, and in CI in qemu, to make sure we have the required components / tools in place as specified.

Reply:
Ok, removed in c154d49d2a9143b60d3d5822d38569af73024ca4. Basic PCR tests would work in osfv QEMU.